webpack.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+        template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
```

Comment:
While you did an amazing job  working on the project requirements. I noticed you disabled eslint in your webpack config file it is against microverse policy to disable linters because linters help us write clean code.